src/task5/task1/Rectangle.java

```diff
@@ -0,0 +1,35 @@
+package task5.task1;
+
+public class Rectangle extends Figure {
+    private int width;
+    private int height;
+
+    public Rectangle(int x, int y, int width, int height) throws RectangleSizeException {
+        super(x, y);
+        validateRectangle(width, height);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    double square() {
+        return width * height;
```

Comment:
?

Reply:
height rename into length?
Or error in another?